src/resources/SearchUsageMetrics/RawMetrics/RawMetrics.ts

```diff
@@ -12,25 +12,44 @@ import {
 export default class RawMetrics extends Resource {
     static baseUrl = `/rest/organizations/${API.orgPlaceholder}/searchusagemetrics/raw/`;
 
+    private addZeroIfRequired = (numberToValidate: number) =>
+        numberToValidate < 10 ? `0${numberToValidate}` : numberToValidate;
```

Comment:
Do you know if there's any native way to format a date to the wanted format? Because it would be easier to maintain something like:

new Date(year,month).format("MM-DD")

Reply:
There is not a direct way to do it (from my quick researches):
You could do that for example:
`new Date(year, month).toLocaleDateString().replace('/','-').slice(-3)`

+ you have to subtract one to the month when you create a date with month (The indexes are between 0-11). Personally, I don't think it's would be better to create a Date object for that instance.

Reply:
Javascript at its best! :P